cypress/integration/dashboard/dashboard_filtering_spec.js

```diff
@@ -0,0 +1,315 @@
+/**
+ * dashboard_filtering test suite description:
+ * 1) Create a new dashboard, populate with visualizations
+ * 2) Set a filter that excludes all data, and check the visualizations for proper updates
+ * 3) Set the existing filter as pinned, re-check the visualizations
+ * 4) Remove the filter, and check the visualizations for proper updates
+ * 5) Create a new dashboard, and populate with a pie graph
+ * 6) Perform apply different filters on the pie graph and check the visualization for proper updates
+ * 7) Remove all filters and ensure the graph reverts to its original design
+ * 8) Test adding another pie graph to the dashboard and applying a filter to both grpahs
+ */
+
+describe('dashboard filtering', () => {
+  before(() => {
+    // Increase the viewport size to prevent pop-up notifications from blocking UI element
+    // E.g. after adding a saved visualization, a notification appears in the bottom right
+    // which could block clicking on a button located behind it.
+    cy.viewport(1900, 1080)
+
+    // TO DO: Setup the indices
+    // cy.request('PUT', 'localhost:9200/.kibana_1', cy.fixture('dashboard/mappings.json'))
+    // cy.request('POST', 'localhost:9200/.kibana_1', cy.fixture('dashboard/data.json.gz'))
+  })
+
+  after(() => {
+    // TO DO: Tear-down the indices
+    // cy.request('DELETE', 'localhost:9200')
```

Comment:
same above, create an issue to track to do

Reply:
Issue has been added (#11)